joinservice/internal/server/server.go

```diff
@@ -100,6 +102,18 @@ func (s *Server) IssueJoinTicket(ctx context.Context, req *joinproto.IssueJoinTi
 		return nil, status.Errorf(codes.Internal, "getting key for stateful disk: %s", err)
 	}
 
+	log.Info("Requesting emergency SSH CA derivation key")
+	ssheCADerivationKey, err := s.dataKeyGetter.GetDataKey(ctx, constants.SSHCAKeySuffix, ed25519.SeedSize)
+	if err != nil {
+		log.With(slog.Any("error", err)).Error("Failed to get emergency SSH CA derivation key")
+		return nil, status.Errorf(codes.Internal, "getting emergency SSH CA derivation key: %s", err)
+	}
+	ca, err := crypto.GenerateEmergencySSHCAKey(ssheCADerivationKey)
+	if err != nil {
+		log.With(slog.Any("error", err)).Error("Failed to derive ssh CA key from derivation key")
+		return nil, status.Errorf(codes.Internal, "generating ssh emergency CA key: %s", err)
+	}
```

Comment:
```suggestion
	sshCAKeySeed, err := s.dataKeyGetter.GetDataKey(ctx, constants.SSHCAKeySuffix, ed25519.SeedSize)
	if err != nil {
		log.With(slog.Any("error", err)).Error("Failed to get seed material to derive SSH CA key")
		return nil, status.Errorf(codes.Internal, "getting emergency SSH CA key seed material: %s", err)
	}
	ca, err := crypto.GenerateEmergencySSHCAKey(sshCAKeySeed)
	if err != nil {
		log.With(slog.Any("error", err)).Error("Failed to derive ssh CA key from seed material")
		return nil, status.Errorf(codes.Internal, "generating ssh emergency CA key: %s", err)
	}
```
similar to my comment for the `crypto` package